src/Aspire.Hosting/Postgres/PostgresBuilderExtensions.cs

```diff
@@ -19,20 +19,39 @@ public static class PostgresBuilderExtensions
     /// </summary>
     /// <param name="builder">The <see cref="IDistributedApplicationBuilder"/>.</param>
     /// <param name="name">The name of the resource. This name will be used as the connection string name when referenced in a dependency.</param>
-    /// <param name="port">The host port used when launching the container. If null a random port will be assigned.</param>
     /// <param name="password">The administrator password used for the container during local development. If null a random password will be generated.</param>
+    /// <param name="port">The host port used when launching the container. If null a random port will be assigned.</param>
     /// <returns>A reference to the <see cref="IResourceBuilder{T}"/>.</returns>
-    public static IResourceBuilder<PostgresServerResource> AddPostgres(this IDistributedApplicationBuilder builder, string name, int? port = null, string? password = null)
+    public static IResourceBuilder<PostgresServerResource> AddPostgres(this IDistributedApplicationBuilder builder, string name, string? password = null, int? port = null)
```

Comment:
I'm wondering whether we should get rid of port.

Reply:
Make it an annotation?

Reply:
We've got the new WithEndpoint with a callback that can mutate the port.

Reply:
Not in this PR